src/Integration.Vsix/Settings/GeneralOptionsDialogControl.xaml.cs

```diff
@@ -18,10 +18,10 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using Microsoft.VisualStudio.Shell;
+using System;
 using System.Windows;
 using System.Windows.Controls;
-using System;
+using Microsoft.VisualStudio.Shell;
```

Comment:
It is an arbitrary decision of @valhristov and I but we have checked the option to put the `System` using first. Options -> Text Editor -> C# -> Advanced, check the `Place 'System' directives first when sorting usings`.